base/bitarray.jl

```diff
@@ -1276,8 +1274,17 @@ function mod(x::Number, B::BitArray)
     y = mod(x, true)
     return fill(y, size(B))
 end
+function broadcast(::typeof(div), B::BitArray, x::Number)
+    T = promote_op(div, Bool, typeof(x))
+    T === Any && return [div(b, x) for b in B]
+    F = Array{T}(size(B))
+    for i = 1:length(F)
+        F[i] = div(B[i], x)
+    end
+    return F
+end
```

Comment:
Cheers, removing this new method and will likewise remove the definition for `mod` that remains (just below) in the relevant PR. (I suppose this change also moves `div`'s promotion behavior in the direction of #19669.) Thanks!